charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       admin/quota: default
 
     profiles:
-      default/allow_experimental_window_functions: "1"
+      {{- merge dict .Values.clickhouse.profiles .Values.clickhouse.defaultProfiles | toYaml | nindent 6 }}
```

Comment:
not sure about the ordering (and if something changes) but should we do default first and then overrides?